superset-frontend/webpack.config.js

```diff
@@ -141,17 +152,6 @@ if (!process.env.CI) {
   plugins.push(new webpack.ProgressPlugin());
 }
 
-// clean up built assets if not from dev-server
-if (!isDevServer) {
-  plugins.push(
-    new CleanWebpackPlugin({
-      dry: false,
-      // required because the build directory is outside the frontend directory:
-      dangerouslyAllowCleanPatternsOutsideProject: true,
-    }),
-  );
-}
-
```

Comment:
Is this no longer needed? I remember this having been a problem before causing lots of accumulated old bundles in the build directory.

Reply:
Webpack 5 has built-in cleaning capabilities. This line replaces the plugin:
![image](https://user-images.githubusercontent.com/15073128/133784969-6b233205-878c-469a-a2b9-2ada8d6da8ad.png)